src/otx/engine/engine.py

```diff
@@ -778,6 +779,80 @@ def explain(
         model.explain_mode = False
         return predict_result
 
+    def benchmark(
```

Comment:
I think it becomes more useful if there is a way to save a result as file. How do you think about it?